aws-rds-dbcluster/aws-rds-dbcluster.json

```diff
@@ -200,8 +200,17 @@
   },
   "allOf": [
     {
-      "required": [
-        "Engine"
+      "oneOf": [
+        {
+          "required": [
+            "Engine"
+          ]
+        },
+        {
+          "required": [
+            "SourceDBClusterIdentifier"
+          ]
+        }
```

Comment:
is it correct? this implies `either or` requirements. According to the docs engine must always be there